src/main/java/bio/terra/service/duos/DuosDao.java

```diff
@@ -103,6 +107,23 @@ public DuosFirecloudGroupModel retrieveFirecloudGroupByDuosId(String duosId) {
     }
   }
 
+  @Transactional(propagation = Propagation.REQUIRED, isolation = Isolation.SERIALIZABLE)
+  public boolean updateFirecloudGroupLastSyncedDate(UUID id, Instant lastSyncedDate) {
+    logger.info("Updating Firecloud group record {} last synced date to {}", id, lastSyncedDate);
+    String sql =
+        """
+            UPDATE duos_firecloud_group
+            SET last_synced_date = :last_synced_date
+            WHERE id = :id
+            """;
+    MapSqlParameterSource params =
+        new MapSqlParameterSource()
+            .addValue("id", id)
+            .addValue("last_synced_date", Timestamp.from(lastSyncedDate));
```

Comment:
Now that we have `Map.of()` I don't use `MapSqlParameterSource` anymore. Maybe I'm missing some benefit that `MapSqlParameterSource` provides?
```suggestion
    var params = Map.of("id", id, "last_synced_date", Timestamp.from(lastSyncedDate));
```

Reply:
Good to know!  Personal preference but I like the clear grouping of parameters to values that `MapSqlParameterSource` provides.